shared/src/vyxal/parsing/Parser.scala

```diff
@@ -3,6 +3,7 @@ package vyxal
 import scala.language.strictEquality
```

Comment:
The package statement at the top is wrong, it should be `package vyxal.parsing`, but we can fix that later sincei t'd involve some (minor) refactoring